vendor/assets/scss/forms/_select.scss

```diff
@@ -26,6 +26,10 @@ $select-radius: $global-radius !default;
   padding: ($form-spacing / 2);
 
   appearance: none;
+  -moz-appearance:none;
+  -webkit-appearance:none;
+  -ms-appearance:none;
```

Comment:
Foundation rely on [Autoprefixer](https://github.com/postcss/autoprefixer) for vendor prefixes and  compatibility. Please consider adding it to your stack instead.